MetaMask/metamask-extension
# UX: Handled empty state of ordered Networks

With the networks drag and drop PR, if the ordered Networks list was empty, extension started crashing. This PR handles the check when orderedNetworks is empty

## **Related issues**

## **Manual testing steps**

- delete node_modules and dist folder
- go to the Version-v11.7.0 branch
- run yarn install & yarn start
-test the switch network , it works fine
- switch to develop
- run yarn install & yarn start
- try switching network ( you will get the error )

To check Networks drag and drop, run extension with `NETWORK_ACCOUNT_DND=1 yarn start` command

## **Screenshots/Recordings**

NA

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
